client/src/components/RelatedProducts/RelatedContext.jsx

```diff
@@ -0,0 +1,50 @@
+import React, {
+  useState, useEffect, useContext, createContext, useMemo,
+} from 'react';
+import axios from 'axios';
+import PropTypes from 'prop-types';
+import { ProductIDContext } from '../../contexts/ProductIDContext';
+
+const RelatedContext = createContext();
+
+export function useRelated() {
+  return useContext(RelatedContext);
+}
+
+export function RelatedProvider({ children }) {
+  const [productInfo, setProductInfo] = useState();
+  const [thumbnail, setThumbnail] = useState();
+  const productId = useContext(ProductIDContext);
+
+  useEffect(() => {
+    axios({
+      method: 'get',
+      url: `http://localhost:3000/products/${productId}`,
```

Comment:
Same comment as with Ivy's - we should be able to leave off the localhost and make the call to `/products/${productId}`. This way we won't have to hunt it down when we're going to deploy